app/views/network_router/_common_new_edit.haml

```diff
@@ -1,123 +1,140 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "networkRouterFormController as vm",
+%form#form_div{'name'          => 'angularForm',
+               'ng-controller' => 'networkRouterFormController as vm',
                'ng-cloak'      => '',
-               "miq-form"      => true,
+               'miq-form'      => true,
                'form-changed'  => true,
-               "model"         => "vm.networkRouterModel",
-               "model-copy"    => 'vm.modelCopy'}
-  = render :partial => "layouts/flash_msg"
+               'model'         => 'vm.networkRouterModel',
+               'model-copy'    => 'vm.modelCopy',
+               'ng-show'       => 'vm.afterGet'}
 
-  %div{"ng-if" => "vm.newRecord"}
-    %h3
-      = _('Network Provider')
-    .form-horizontal
-      .form-group{"ng-class" => "{'has-error': angularForm.ems_id.$invalid}"}
-        %label.col-md-2.control-label
-          = _('Network Manager')
-        .col-md-8
-          %select{'id'         => 'ems_id',
-                  'name'       => 'ems_id',
-                  'ng-change'  => "vm.filterNetworkManagerChanged(vm.networkRouterModel.ems_id)",
-                  'ng-model'   => "vm.networkRouterModel.ems_id",
-                  'ng-options' => 'ems.id as ems.name for ems in vm.ems',
-                  'pf-select'  => true,
-                  'required'   => ''}
-            %option{:value => ""}
-              = "<#{_('Choose')}>"
-          %span.help-block{"ng-show" => "angularForm.ems_id.$error.required"}
-            = _('Required')
+  = render :partial => 'layouts/flash_msg'
 
   %h3
-    = _('Basic Information')
+    = _('Network Management Provider')
+
+  .form-horizontal
+    .form-group
```

Comment:
This should be `.form-group{'ng-class' => '{"has-error": angularForm.ems_id.$invalid}'}` the selectpicker and Required text turn red.

Reply:
Good catch! Done, thanks.